src/livePreview/editButton/editButton.ts

```diff
@@ -6,11 +7,20 @@ import {
     IClientUrlParams,
     IConfigEditButton,
     IEditButtonPosition,
+    ILivePreviewMode,
+    ILivePreviewModeConfig,
     ILivePreviewWindowType,
     IStackDetails,
 } from "../../types/types";
 import livePreviewPostMessage from "../eventManager/livePreviewEventManager";
 
+effect(function handleWindowTypeChange() {
+    Config.get().windowType;
```

Comment:
This line is a fix to tell the effect to rerun this function every time we update the value. Let's add a comment explaining that.